playbooks/configure-apt-sources.yml

```diff
@@ -119,48 +119,129 @@
         - not apt_artifact_enabled | bool
 
   tasks:
-    - name: Sync artifact files (all)
-      copy:
-        src: "{{ item }}"
-        dest: "/etc/openstack_deploy/group_vars/all/{{ item | basename }}"
-      with_items:
-        - files/apt.yml
-      run_once: true
-      delegate_to: localhost
-      when:
-        - apt_artifact_found | bool
-        - apt_artifact_enabled | bool
+    - name: Deploy apt artifacts
+      block:
+        - name: Backup the original sources list
+          command: "cp /etc/apt/sources.list /etc/apt/sources.list.original"
```

Comment:
Is there a reason for the use of command rather than the source which used copy?

```
    - name: Backup the original sources file
      copy:
        src: /etc/apt/sources.list
        dest: /etc/apt/sources.list.original
        remote_src: yes
        force: no
```

Reply:
yes I only want to create the original file once bu I see setting the force flag should do what I need.